docs/rules/new-cap.md

```diff
@@ -8,77 +8,135 @@ var friend = new Person();
 
 ## Rule Details
 
-This rule is aimed at helping to distinguish regular functions from constructor functions. As such, it warns whenever it sees `new` followed by an identifier that isn't capitalized or whenever it sees capitalized function called directly without `new` operator.
+This rule requires constructor names to begin with a capital letter. Certain built-in identifiers are exempt from this rule. These identifiers are:
 
-The following patterns are considered problems:
+* `Array`
+* `Boolean`
+* `Date`
+* `Error`
+* `Function`
+* `Number`
+* `Object`
+* `RegExp`
+* `String`
+* `Symbol`
+
+## Options
+
+This rule has an object option:
+
+* `"newIsCap": true` (default) requires all `new` operators to be called with uppercase-started functions.
+* `"newIsCap": false` disables checks for all `new` operators to be called with uppercase-started functions.
```

Comment:
Yes, good to keep parallel grammar. Seems worthwhile to be explicit what `false` value allows, using the same term as in the exception options, hopefully to make it clear how they relate.

`"newIsCap": false` allows new operators to be called with lowercase-started or uppercase-started functions.